ieee.csl

```diff
@@ -39,6 +43,9 @@
     </terms>
   </locale>
   <!-- Macros -->
+  <macro name="status">
+    <text variable="status" text-case="capitalize-first" suffix="" font-weight="bold"/>
```

Comment:
In the styleguide, I'm seeing 
> [8] E. H. Miller, “A note on reflector arrays,” IEEE Trans. Antennas Propag..., to be published.*

which would suggest not using bold and not capitalizing the first letter (which seems weird given that comma before it anyway). Any reason for those two choices?

Reply:
They are style preferences, my preferences. I would like to emphasize that the paper is in print or about to be published. If you do not think they are useful, you can remove them, of course.